src/PhpSpec/Container/ServiceLocator.php

```diff
@@ -11,15 +11,16 @@
  * file that was distributed with this source code.
  */
 
-namespace PhpSpec;
+namespace PhpSpec\Container;
 
+use Interop\Container\ContainerInterface;
 use InvalidArgumentException;
 
 /**
  * The Service Container is a lightweight container based on Pimple to handle
  * object creation of PhpSpec services.
  */
-class ServiceContainer
+class ServiceLocator implements ContainerInterface
```

Comment:
IMO, the name ``PhpSpec\ServiceContainer`` should be kept for now, until we know what extensions should look like, to minimize changes